app/controllers/api/v1/instances/peers_controller.rb

```diff
@@ -8,7 +8,7 @@ class Api::V1::Instances::PeersController < Api::BaseController
 
   def index
     expires_in 1.day, public: true
-    render_with_cache(expires_in: 1.day) { Account.remote.domains }
+    render_with_cache(expires_in: 1.day) { Instance.pluck(:domain) }
```

Comment:
Peers are used for discovery and pre-emptively blocking something means that something exists. We didn't exclude blocked domains from here before so I see little difference.

Reply:
I think blocked domains didn't use to show up, since blocking a domain would delete the account records. An example is gab, it doesn't exist anymore, yet people still routinely block it. Some people also try to assert how well-connected it is/was, and use the peers endpoint for that. This change would make it appear connected to anyone who blocked it.